### PR TITLE
Update to use official sphero.js module.

### PR DESCRIPTION
**Pull request by edgarsilva**
Updated adaptor, driver, tests and examples to use the newly released sphero.js module from orbotix.

### CODE REVIEW

File: examples/color/fluent-color.js [hunk 1]
```diff
@@ -6,10 +6,14 @@ Cylon
...
   .connection("sphero", { adaptor: "sphero", port: "/dev/rfcomm0" })
   .device("sphero", { driver: "sphero" })
-  .on("ready", function(bot) {
-    setInterval(function() {
-      bot.sphero.setRGB(Math.floor(Math.random() * 100000));
-    }, 1000);
+  .on("ready", function(my) {
+    every((1).second(), function() {
+      // We tell spherot o change the color of its
+      // RGB LED to a random value.
```
Comment (stewart) [inline examples/color/fluent-color.js]: Can you correct the spelling errors here and in the related files?

File: package.json [hunk 2]
```diff
@@ -16,15 +16,16 @@
...
+    "mocha":      "~2.2.5",
+
+    "eslint":     "0.22.1"
   },
 
   "dependencies": {
-    "cylon":               "~1.0.0",
-    "hybridgroup-spheron": "~0.3.0"
+    "cylon":  "~1.0.0",
+    "sphero": ">=0.3.0"
```
Comment (stewart) [inline package.json]: We probably don't want to use a greedy version specifier here? Locking to `0.3.0` should be fine.

File: spec/lib/adaptor.spec.js [hunk 3]
```diff
@@ -57,7 +57,16 @@ describe("Adaptor", function() {
...
 
-    context("if no pin is specified", function() {
+    context("if opts are provided", function() {
+      it("throws an error", function() {
+        var fn = function() { return new Adaptor(); };
+        expect(fn).to.throw(
+          "No port specified for Sphero adaptor 'undefined'. Cannot proceed"
+        );
+      });
+    });
```
Comment (stewart) [inline spec/lib/adaptor.spec.js]: Isn't this a duplicate of the test directly below it?

Comment (edgarsilva) [inline spec/lib/adaptor.spec.js]: Nope, one tests if only port is not provided, the other tests when no `opts` at all are provided. Behaviour varies slightly in the code.

**Pull request merged**